### PR TITLE
OSOE-62: setupRecommended and getNugetPath

### PR DESCRIPTION
**Pull request by sarahelsaig**
OSOE-62

### CODE REVIEW

File: recommended-setup.js [hunk 1]
```diff
@@ -114,6 +121,67 @@ function setupRecommendedScssAndJsTasksAndCopyAssets(
...
+        add('assets', 'copy');
+    }
+
+    gulp.task('default', gulp.parallel(...build));
+    gulp.task('clean', gulp.parallel(...clean));
+
+    return this;
+}
+
+function getNugetPath() {
```
Comment (0liver) [inline recommended-setup.js:162]: @DAud-IcI What exactly did you need this for again? I don't quite grasp all of what you've added in this PR.

Comment (sarahelsaig) [reply recommended-setup.js:162]: It's for getting the physical location of a nuget package in the file system. Similar to the [package path property in MSBuilld](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty). It was necessary to get packaged assets from the NuGet package and pass it to gulp, e.g. for resolving SCSS files in Lombiq.BaseTheme, see [here](https://github.com/Lombiq/Orchard-Base-Theme/blob/7723404db46cce753675ad9137d503d0556f9e6f/Lombiq.BaseTheme.Samples/Gulpfile.js#L33-L37))

Comment (0liver) [reply recommended-setup.js:162]: Why do we need to know the path of a NuGet package for SCSS compilation? What's the use case? Are you trying to access files from `Lombiq.BaseTheme` in `Lombiq.BaseTheme.Samples`? Why?

Comment (sarahelsaig) [reply recommended-setup.js:162]: If you want to make a theme utilizing `Lombiq.BaseTheme` as a NuGet package. the localtion of the base theme's scss files need to be told to the sass compiler.

Comment (0liver) [reply recommended-setup.js:162]: I see.

**Pull request merged**